tsconfig.json

```diff
@@ -7,7 +7,8 @@
         "jsx": "react",
         "baseUrl": ".",
         "resolveJsonModule": true,
-        "importHelpers": true
+        "importHelpers": true,
+        "importsNotUsedAsValues": "error"
```

Comment:
this option was [deprecated](https://www.typescriptlang.org/tsconfig#importsNotUsedAsValues) in of favor `verbatimModuleSyntax `